service-api/module/Application/config/module.config.php

```diff
@@ -148,6 +148,26 @@
                     ],
                 ],
             ],
+            'get_kbv_questions' => [
+                'type'    => Segment::class,
+                'options' => [
+                    'route'    => '/cases[/:uuid]/kbv-questions',
+                    'defaults' => [
+                        'controller' => Controller\IdentityController::class,
```

Comment:
Should probably think about splitting out into multiple controllers, e.g. adding these endpoints to a new `KbvController`

Reply:
Created ticket for this